<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#background {
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #f74780, stop:1 #fa8fb1);
}
QWidget#main_container {
	background-color: #ffe4ec;
	border-radius: 20px;
}

QWidget#float_window {
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #f74780, stop:1 #fa8fb1);
	border-radius: 20px;
}</string>
  </property>
  <widget class="QWidget" name="background">
   <widget class="QWidget" name="main_container" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1251</width>
      <height>701</height>
     </rect>
    </property>
    <widget class="QWidget" name="float_window" native="true">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>80</y>
       <width>481</width>
       <height>521</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="container" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1251</width>
       <height>701</height>
      </rect>
     </property>
     <widget class="QPushButton" name="btn_edit">
      <property name="geometry">
       <rect>
        <x>1130</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>EDIT</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
